packages/neo4j-driver/test/driver.test.js

```diff
@@ -26,7 +26,9 @@ import {
 } from '../../bolt-connection/lib/pool/pool-config'
 import { ServerVersion, VERSION_4_0_0 } from '../src/internal/server-version'
 import testUtils from './internal/test-utils'
-import { json } from 'neo4j-driver-core'
+import { json, internal } from 'neo4j-driver-core'
+
+const { bookmark } = internal
```

Comment:
can you write `import { json, { bookmark } } from 'neo4j-driver-core'`? :P

Reply:
Unfortunately, no. 

I've put everything from core which is not supposed to be externalised by the driver in the `internal` package for making easier to realize it should not be externalised. 

It will be good with. could do `import { internal: { bookmark } } from 'neo4j-driver-core`, but unfortunately It can't be done. 😢